workspaces/default/themes/base/partials/auth/forms/register/basic-auth.html

```diff
@@ -76,9 +76,10 @@
               .text(response.fields[field])
           }
         } else {
+          const uiErrorMessage = data && data.message && typeof data.message  === "string" ? data.message : fallbackErrorMessage;
```

Comment:
I like the improvement of ensuring the message is a string before rendering it so we don't ever see `[Object object]`. The response in the screenshot shows we have some information from this error that we can render to help the user out a little more. What do you think about a function that works like this:

```
function getErrorMessage(error) {
  const defaultMessage = 'An unexpected error occurred. Please try again.'

  const isObject = o => o && typeof o === 'object'

  if (!error) { return defaultMessage }

  if (error.message && typeof error.message === 'string') { return error.message }

  if (isObject(error.message)) {
    if (isObject(error.message.error)) {

      // Get message for errors with problems sending emails:
      // {
      //   message: {
      //     error: { emails: { 'noreply@konghq.com': 'Error sending email'}, count: 1 },
      //     sent: { emails: {}, count: 0 }
      //   }
      // }
      if (isObject(error.message.error.emails)) {
        return Object.entries(error.message.error.emails).map(([k, v]) => {
          return `${v} from ${k}`
        }).join('\n')
      }
    }
  }

  return defaultMessage
}
```
e.g.,
```
const message = getErrorMessage({
  message: {
    error: { emails: { 'noreply@konghq.com': 'Error sending email'}, count: 1 },
    sent: { emails: {}, count: 0 }
  }
})

// Should output: 'Error sending email from noreply@konghq.com\n'
console.log(message)
```

Then we can add more cases to the `getErrorMessage()` function in the future if we want to create more friendly messages for other things.